ProyectoSemestral3/Assets/Scripts/PlayerScripts/PlayerMovement.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private CharacterController _controller;
+    [SerializeField] private Transform cam;
+    [SerializeField] private float _speed = 6f;
+    private float _runningSpeed = 12f;
+    private float _speedDecrement = 6f;
+    [SerializeField] private float _turnTime = 0.1f;
+    [SerializeField] float _directionDeadZone = 0.1f;
+    float _turnVelocity;
```

Comment:
Have all `[SerializeField] private` together, all private together, and check that all variables have default value, `_` and access modifier.